src/components/Article.vue

```diff
@@ -43,12 +48,30 @@ export default {
               });
             }
           }).catch(err => console.log(err));
+    },
+    initializeDisqus : function () {
+      /**
+       *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+       *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
+
+      var disqus_config = function () {
+        this.page.url = "https://simplecoding.dev/articles";  // Replace PAGE_URL with your page's canonical URL variable
```

Comment:
should it be "https://simplecoding.dev/articles/:slug"; ? please check the documentation